src/store/checkout-store.js

```diff
@@ -32,6 +34,23 @@ class CheckoutStore extends Store {
 
 const store = new CheckoutStore();
 
+function updateCheckoutFrequency(equipment){
+    equipment.forEach(equip => {
+        let result = readAddress(equip.address);
+        let model = null;
+        if (result.type === 'item'){
+            let item = ItemStore.getItemByAddress(equip.address);
+            model = ModelStore.getModelByAddress(item.modelAddress);
+        }else model = ModelStore.getModelByAddress(equip.address);
+        if(!model.frequency){
```

Comment:
Doesn't need to be added in the action itself (because it would always be zero), but we could/should set the frequency of the model to 0 in the action handler.